client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -20,6 +30,57 @@ import EventSponsorsForm from './EventSponsorsForm';
 import EventVenueForm from './EventVenueForm';
 import { EventFormProps, fields, EventFormData } from './EventFormUtils';
 
+interface IntegrationInfoData {
+  isAuthenticated: boolean | null | undefined;
+  hasCalendar: boolean;
+}
+
+const IntegrationInfo = ({
+  isAuthenticated,
+  hasCalendar,
+}: IntegrationInfoData) => {
+  const infoData = ({ isAuthenticated, hasCalendar }: IntegrationInfoData) => {
+    if (isAuthenticated) {
+      if (hasCalendar) {
+        return {
+          Icon: InfoIcon,
+          text: 'Instance is authenticated with calendar api, and chapter has created calendar. Chapter will attempt creating event in the calendar.',
+        };
+      }
+      return {
+        Icon: WarningTwoIcon,
+        text: "Instance is authenticated with calendar api, but chapter doesn't have calendar created. Event will not be created in the calendar.",
+      };
+    }
+
+    const isBroken = isAuthenticated === null;
+    if (isBroken) {
+      if (hasCalendar) {
+        return {
+          Icon: WarningTwoIcon,
+          text: 'Chapter has calendar created, but calendar integration is not working. Event will not be created in the calendar.',
+        };
+      }
+      return {
+        Icon: WarningTwoIcon,
+        text: 'Calendar integration is not working, and chapter does not have calendar created. Event will not be created in calendar.',
+      };
+    }
+    return {
+      Icon: WarningTwoIcon,
+      text: 'Instance is not authenticated with calendar api. Automatic creation of events in calendar is not possible.',
+    };
```

Comment:
Do we need to distinguish between these three cases? I think it makes sense to handle `hasCalendar` separately, but if Chapter's tokens are missing or invalid, the fix is simply to reauthenticate.

With that in mind, how about this

> Chapter is not authenticated with Google Calendar. Go to `link to calendar dash` to authenticate.

> Chapter is not authenticated with Google Calendar and this chapter does not have a calendar. Go to `link to calendar dash` to authenticate. If you wish to create a calendar, visit the chapter dashboard.

Reply:
I'm a bit assuming that person seeing these warnings not necessarily has privileges to (re-)authenticate. Otherwise the alert on the top would be catching the attention to re-authenticate.